rita_common/src/tunnel_manager/neighbor_status.rs

```diff
@@ -49,6 +49,7 @@ pub fn update_neighbor_status() {
                 id: *id,
                 shaper_speed: lowest_shaper_speed,
                 enforced,
+                neighbor_name: id.wg_public_key.to_string(),
```

Comment:
Considering how this value is filled here this seems to be a placeholder for the ops side. I don't see a reason to add the field here rather than in operator tools then.